src/components/calculator.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+export const Calc = () => {
```

Comment:
- [ ] As it is among the project requirements, all the components including this one should be `class components` so please refactor your codes here in order to use the class component method  **just like you did in the App.js**😉 🏃

Reply:
ok, no problem i'm gonna do it!